test/src/subnet/main.cpp

```diff
@@ -74,55 +91,65 @@ using logger_top=cadmium::logger::multilogger<log_messages, global_time>;
 /********************************************/
 /****** APPLICATION GENERATOR *******************/
 /********************************************/
-string input_data = "test/data/subnet/subnet_input_test.txt";
-const char * i_input_data = input_data.c_str();
```

Comment:
a pointer variable should start with p_<variable_name>

Reply:
changed after review

Reply:
verified